roles/pulp_devel/templates/alias.bashrc.j2

```diff
@@ -159,6 +159,10 @@ pbindings(){
     lang="${2:-python}"
     comp="${1:-pulpcore}"
 
+    if [[ -z "${PULP_V3_API_ROOT}" ]]; then
+      PULP_V3_API_ROOT=$(django-admin shell -c "from django.conf import settings;print(settings.V3_API_ROOT)")
+    fi
```

Comment:
you need it after `workon pulp`

Reply:
I looked into doing that, but I don't see how it's possible because `workon pulp` actually uses `mkvirtualenv` which is invoked directly. Do have an idea of where I can put this?

Reply:
https://github.com/pulp/pulp_installer/blob/main/roles/pulp_devel/templates/alias.bashrc.j2#L171

Reply:
Lol well that's easy. Thanks!